src/redux/books/booksSlice.js

```diff
@@ -0,0 +1,37 @@
+const ADD_BOOK_ITEM = 'ADD_BOOK_ITEM';
+const REMOVE_BOOK_ITEM = 'REMOVE_BOOK_ITEM';
+
+const addBookItem = (book) => ({
+  type: ADD_BOOK_ITEM,
+  payload: book,
+});
+
+const removeBookItem = (bookId) => ({
+  type: REMOVE_BOOK_ITEM,
+  payload: bookId,
+});
+
+const initialState = {
+  books: [],
+};
+
+const booksReducer = (state = initialState, action) => {
+  switch (action.type) {
+    case ADD_BOOK_ITEM:
+      return {
+        ...state,
+        books: [...state.books, action.payload],
+      };
+    case REMOVE_BOOK_ITEM:
+      return {
+        ...state,
+        books: state.books.filter((book) => book.id !== action.payload),
+      };
+    default:
+      return state;
+  }
+};
+
+export {
+  ADD_BOOK_ITEM, REMOVE_BOOK_ITEM, addBookItem, removeBookItem, booksReducer,
+};
```

Comment:
- [ ] Your implementation of the book reducer and actions do not follow the requirements. You are required to import `createSlice` from the redux toolkit and use it to set up your reducer for the book. Kindly refactor this file to fit this requirement. You can refer to [this documentation](https://redux-toolkit.js.org/api/createSlice).

Reply:
booksSlice.js file refactored to import createSlice from redux toolkit✔